### PR TITLE
refactor(modules): Import angular hint modules

### PR DESCRIPTION
**Pull request by welbornio**
@btford working branch

### CODE REVIEW

**Comment by welbornio** [comment]
not ready for merge yet

**Comment by btford** [comment]
:clap:

Thanks so much!

**Comment by welbornio** [comment]
references #69

File: test/events.spec.js [hunk 1]
```diff
@@ -26,14 +26,28 @@ describe('hintEvents', function() {
   });
 
   // TODO: implement this
-  xit('should log a message if the path to handle an ng-event is not found', function() {
+  it('should log a message if the path to handle an ng-event is not found', function() {
     var elt = angular.element('<button ng-click="a.b.c()">Fake Increment</button>');
```
Comment (welbornio) [inline test/events.spec.js]: This is code to be extracted into a separate PR
